wa/utils/log.py

```diff
@@ -148,6 +149,13 @@ def dedent():
     _indent_level -= 1
 
 
+@contextmanager
+def indentcontext():
+    indent()
+    yield
+    dedent()
```

Comment:
Should this be in a try/finally block in case of errors?

Reply:
No, this is ensured by the context machinery.

Reply:
OK, I was wrong, `contextmanager` decorator *wont* ensure that by default. Will fix.